game/match/match.go

```diff
@@ -1801,7 +1801,7 @@ func (m *Match) handleAdminMesseges(message string, user db.User) {
 	// /mana red 4 - add 4 fire mana
 	// /mana n - add 1 nature mana
 	// /mana imageID 3 - add 3 mana of that specific card
-	if string(message[0:5]) == "/mana" {
+	if string(message[0:6]) == "/mana " {
 
 		msgParts := strings.Split(message, " ")
```

Comment:
This could probably be moved outside of the individual commands and used to test for the first word so that we don't need to have the `len(message) < 7` check above but instead check that it's at least 1 word long in case someone adds a new command with more than 7 characters in the future without updating the length check

Reply:
ok, changed it